ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -45,14 +45,44 @@ tests:
   optional: true
   steps:
     cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-noo-bundle
+    env:
+      OO_CHANNEL: latest
+      OO_INSTALL_NAMESPACE: openshift-netobserv-operator
+      OO_PACKAGE: netobserv-operator
+      OO_TARGET_NAMESPACES: '!all'
     test:
     - as: e2e-operator
-      commands: make test
+      cli: latest
+      commands: |
+        export PATH=$PATH:$HOME
+
+        if ! which kubectl; then
+          ln -s "$(which oc)" ${HOME}/kubectl
+        fi
+
+        export CSV=$(oc get csv -n openshift-netobserv-operator | grep -E "net.*observ" | awk '{print $1}')
+        USER=netobserv VERSION=main make set-agent-image
+        USER=netobserv VERSION=main make set-flp-image
+        USER=netobserv VERSION=main make set-plugin-image
```

Comment:
I suppose that's a preparatory work for future capabilities to pass custom image arguments ?

Reply:
Sorry, I missed this.
Yes, that's right. Once we add configs for other operands, we'd have to find a way to such that it deploys custom images that's build during the test.